app/assets/components/tagify/.bower.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
my thought  if there is any way to avoid adding this dependency manually? Can't we find a gem or a similar gem for tags?